web/api/formatters/charts2json.c

```diff
@@ -4,7 +4,7 @@
 
 // generate JSON for the /api/v1/charts API call
 
-static inline const char* get_release_channel() {
+inline const char* get_release_channel() {
```

Comment:
Since this function became externalized we should remove the `inline` altogether otherwise we'll have a lot of linking problems.

Reply:
I will change this -- I will remove the external, and return the static for now since it is not used. If need it will be applied in the future